sample/src/main/res/layout/fragment_editable_list_view.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    style="@style/SampleScrollView">
+
+    <LinearLayout
+        android:id="@+id/content_layout"
+        style="@style/SampleSectionContainer">
+
+        <uk.gov.hmrc.components.atom.heading.Heading5 style="@style/PlaceholderText" />
+
+        <uk.gov.hmrc.components.organism.primary.PrimaryCardView
+            android:id="@+id/headline_placeholder"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_margin="@dimen/hmrc_spacing_16"
+            app:headline="@string/headline_placeholder_headline"
+            app:title="Your Company Benefits">
```

Comment:
The placeholder should just be the basic bare boned component, how it comes, out of the box.. so we shouldn't put it on a card cause the card doesn't come built in to the component. The examples section is for that!

Because this component doesn't have a background colour built in, we can use `<FrameLayout style="@style/PlaceholderContainer">` to give it a temporary background for this placeholder example.